cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -122,12 +122,43 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
-    {# Google Tag Manager #}
+
+    <!-- Google Optimize page-hiding snippet -->
+    <style>
```

Comment:
yeah, just seems like we are going from analytics change to the next analytics change, without having a full understanding about the previous change. Is there going to be a change in another month because we don't understand something about this change?